code/game/area/areas/ruins/lavaland.dm

```diff
@@ -3,6 +3,9 @@
 
 /area/ruin/powered/beach
 
+/area/ruin/powered/Phone_booth
```

Comment:
```suggestion
/area/ruin/powered/lavaland_phone_booth
```

Caps in an area path, make this explicitly the lavaland area. Since the area is powered you can remove the APC.